docs/source/tensor_attributes.rst

```diff
@@ -213,7 +213,8 @@ non-None device argument.  To globally change the default device, see also
 
 .. note::
    For legacy reasons, a device can be constructed via a single device ordinal, which is treated
-   as a cuda device.  This matches :meth:`Tensor.get_device`, which returns an ordinal for cuda
+   as a currently available device type (i.e. "cuda" if cuda is available, "xpu" if xpu is available).
```

Comment:
Given that we're going to have a few features relying on it as the default (pin memory for example), I think we should make "accelerator" a public concept in this doc.
I think we can:
- Have one paragraph in https://pytorch.org/docs/stable/torch.html that introduces the concept of accelerator and lists all the current ones (based on the list in c10).
- Have other docs like this one just mention the "current accelerator" with a link to the paragraph above.

WDYT?

Reply:
Sounds good. Maybe we can take below RFC together with this PR, to figure out an accelerator mechanism both in frontend and backend? https://github.com/pytorch/pytorch/issues/128403

@guangyey let's have a talk next week.

Reply:
It is great to me~